iterative/utils/analytics.go

```diff
@@ -22,8 +26,15 @@ import (
 	"golang.org/x/crypto/scrypt"
 )
 
+const (
+	Timeout  = 5 * time.Second
+	Endpoint = "https://telemetry.cml.dev/api/v1/s2s/event?ip_policy=strict"
+	Token    = "s2s.jtyjusrpsww4k9b76rrjri.bl62fbzrb7nd9n6vn5bpqt"
+)
+
```

Comment:
I do not get the `ip_policy=strict` issue/craziness. It was already set?!

Reply:
It was because of [this conversation](https://iterativeai.slack.com/archives/C0249LW0HAQ/p1650371913116649?thread_ts=1649179052.132839&cid=C0249LW0HAQ).

Reply:
Nevermind